### PR TITLE
New package: Samsung.MagicDoctor version 5.0.3.1

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/111980&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/s/Samsung/MagicDoctor/5.0.3.1/Samsung.MagicDoctor.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Samsung.MagicDoctor
+PackageVersion: 5.0.3.1
+Installers:
+- InstallerUrl: https://org.downloadcenter.samsung.com/downloadfile/ContentsFile.aspx?CDSite=US&CttFileID=1908720&CDCttType=SW&ModelType=C&ModelName=NP-Q1U/P01/SEA&VPath=SW/200804/20080422161039281_Magic_Doctor_5.0.3.1.zip
```
Comment (Trenly) [inline manifests/s/Samsung/MagicDoctor/5.0.3.1/Samsung.MagicDoctor.installer.yaml:7]:
```suggestion
- InstallerUrl: https://downloadcenter.samsung.com/content/SW/200804/20080422161039281_Magic_Doctor_5.0.3.1.zip
```

File: manifests/s/Samsung/MagicDoctor/5.0.3.1/Samsung.MagicDoctor.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Samsung.MagicDoctor
+PackageVersion: 5.0.3.1
```
Comment (Trenly) [inline manifests/s/Samsung/MagicDoctor/5.0.3.1/Samsung.MagicDoctor.installer.yaml:5]:
```suggestion
PackageVersion: 5.00
```

File: manifests/s/Samsung/MagicDoctor/5.0.3.1/Samsung.MagicDoctor.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Samsung.MagicDoctor
+PackageVersion: 5.0.3.1
```
Comment (Trenly) [inline manifests/s/Samsung/MagicDoctor/5.0.3.1/Samsung.MagicDoctor.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 5.00
```

File: manifests/s/Samsung/MagicDoctor/5.0.3.1/Samsung.MagicDoctor.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: Samsung.MagicDoctor
+PackageVersion: 5.0.3.1
```
Comment (Trenly) [inline manifests/s/Samsung/MagicDoctor/5.0.3.1/Samsung.MagicDoctor.yaml:5]:
```suggestion
PackageVersion: 5.00
```

File: manifests/s/Samsung/MagicDoctor/5.0.3.1/Samsung.MagicDoctor.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Samsung.MagicDoctor
+PackageVersion: 5.0.3.1
+PackageLocale: en-US
+Publisher: Samsung
```
Comment (Trenly) [inline manifests/s/Samsung/MagicDoctor/5.0.3.1/Samsung.MagicDoctor.locale.en-US.yaml:7]:
```suggestion
Publisher: Samsung Electronics CO., LTD.
```

**Review by Trenly — CHANGES_REQUESTED** [review]
image

**Comment by stephengillie** [comment]
> Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\s\Sa***ung\MagicDoctor\5.0.3.1', expected 'manifests\s\Sa***ung\MagicDoctor\5.00'

**Comment by stephengillie** [comment]
>  Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\s\Samsung\MagicDoctor\5.0.3.1', expected 'manifests\s\Samsung\MagicDoctor\5.00'

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

**Comment by stephengillie** [comment]
image

**Comment by upintheairsheep** [comment]
> image

https://answers.microsoft.com/en-us/windows/forum/all/error-1275-this-driver-has-been-blocked-from/71e7cda2-7fa1-4d76-9836-8acc227b1a2a

**Pull request closed without merging**